addons/web_dashboard/controllers/main.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+
+from openerp import http, _
+from openerp.http import request
+from openerp.exceptions import RedirectWarning
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class Dashboard(http.Controller):
+
+    def get_users_list(self):
+        users = request.env['res.users'].search([('login_date', '=', False)], order="create_date desc")
+        pending = users.filtered(lambda u: u.signup_valid)
+        expired = users - pending
+        return {'expired': zip(expired.mapped('id'), expired.mapped('login')), 'pending': zip(pending.mapped('id'), pending.mapped('login'))}
+
+    @http.route('/dashboard/info', type='json', auth='user')
+    def get_info(self, **kw):
+        installed_apps = request.env['ir.module.module'].search_count([('application', '=',  True), ('state', 'in', ['installed', 'to upgrade', 'to remove'])])
+        active_users = request.env['res.users'].search_count([('active', '=', True), ('login_date', '!=', False)])
+        planners = request.env['web.planner'].search_read([], ['progress', 'menu_id'], order='name')
+        return {
+            'apps': {
+                'installed_apps': installed_apps
+            },
+            'users_info': {
+                'active_users': active_users,
+                'users': self.get_users_list(),
+                'template_user_id': request.env['ir.config_parameter'].get_param('auth_signup.template_user_id'),
+                'user_form_id': request.env['ir.model.data'].xmlid_to_res_id("base.view_users_form")
+            },
+            'planner': {
+                'planners': planners,
+                'overall_progress': sum([p['progress'] for p in planners]) / (len(planners) or 1)
+            },
+            'gift': {
+                'user_email': request.env.user.email
+            }
+        }
+
+    @http.route('/dashboard/create_users', type='json', auth='user')
+    def create_user(self, email_adderesses, optional_message=""):
+        Users = request.env['res.users']
+        email_adderesses = map(str.strip, email_adderesses)
```

Comment:
I'm not getting traceback on user creation from dashboard. Can you tell me steps to reproduce ??